cli/xlsx2json.py

```diff
@@ -0,0 +1,250 @@
+                    has_notnull = True
+                data[header] = self.trim_value(self.cast_value(
+                    cell.value, self.xls_conf[worksheet].get(CAST_KEY_NAME, {}).get(header)
+                ))
+
+            if has_notnull:
+                data['row_num'] = self.row_offset[worksheet]
+                rows.append(data)
+
+            # no data on this row, continue to next
+            self.row_offset[worksheet] += 1
+
+        return rows
+
+    def json(self, output_json_file):
+        json_data = {}
+        for title in self.xls_conf[WORKSHEETS_KEY_NAME]:
+            json_data[self.xls_conf[WORKSHEETS_KEY_NAME][title]] = []
+            self.active_worksheet = title
+            for row in self.get_rows():
+                # Remove the row number
+                row.pop('row_num')
+                # remove any None
+                json_data[self.xls_conf[WORKSHEETS_KEY_NAME][title]].append(
+                    {self.translate_header(title, k): v for k, v in row.items() if v is not None}
+                )
+
+        with open(output_json_file, 'w') as open_file:
+            json.dump(json_data, open_file, cls=DateTimeEncoder)
+
+    def translate_header(self, title, header):
+        if header in self.xls_conf[title].get(REQUIRED_HEADERS_KEY_NAME, {}):
+            return self.xls_conf[title][REQUIRED_HEADERS_KEY_NAME][header]
+        if header in self.xls_conf[title].get(OPTIONAL_HEADERS_KEY_NAME, {}):
+            return self.xls_conf[title][OPTIONAL_HEADERS_KEY_NAME][header]
+        return header
+
+
+def create_xls_template_from_yaml(xls_filename, conf_filename):
+    """
+    Create a XLS empty template with the expected worksheet and header based on the configuration provided
+    """
+    with open(conf_filename, 'r') as conf_file:
+        xls_conf = yaml.full_load(conf_file)
+    workbook = Workbook()
+    workbook.remove(workbook.active)
+
+    for title in xls_conf[WORKSHEETS_KEY_NAME]:
+        # Create worksheet
+        workbook.create_sheet(title)
+        # For each Worksheet create the header row
+        header_row = xls_conf[title].get(HEADERS_KEY_ROW, 1)
+        header_col_index = 0
+        for header_name in xls_conf[title].get(REQUIRED_HEADERS_KEY_NAME, []):
+            header_col_index += 1
+            cell = workbook[title].cell(column=header_col_index, row=header_row, value=header_name)
+            cell.font = openpyxl.styles.Font(bold=True)
+        for header_name in xls_conf[title].get(OPTIONAL_HEADERS_KEY_NAME, []):
+            header_col_index += 1
+            workbook[title].cell(column=header_col_index, row=header_row, value=header_name)
+    workbook.save(xls_filename)
+
+
+def main():
+    arg_parser = argparse.ArgumentParser(
+        description='Compare the sample name in the VCF file and the one specified in the metadata.')
+    arg_parser.add_argument('--metadata_json', required=True, dest='metadata_json', help='EVA metadata json file')
```

Comment:
```suggestion
    arg_parser.add_argument('--metadata_json', required=True, dest='metadata_json', help='Path to output EVA metadata json file')
```
(or something similar to specify this one is the output destination)